todo_app/templates/index.html

```diff
@@ -38,6 +38,7 @@ <h2>Items</h2>
     </div>
     <div class="change-item-form">
       <input type="submit" name="changeItem" value="Save Item">
+      <input type="submit" name="changeItem" value="Complete Item">
```

Comment:
This approach works, but it can be a little fiddly as a user to be typing in ids everywhere. Would it be nicer to put a form per card, so the user just clicks the corresponding button?
E.g. for `delete` functionality, that might look like changing the item loop to be more like below (note the `hidden` keyword to save the users needing to think about the inputs)
```HTML
        {% for item in items %}
          <li>{{item.id}} - {{item.status}} - {{item.name}}</li>
          
          <form action="/changeItem" method="post" class="change-item-form">
            <input type="numeric" name="itemID" id="itemID" size="1" value="{{item.id}}" hidden>
            <input type="submit" name="changeItem" value="Delete Item">
          </form>
        {% endfor %}
```

Reply:
Would definitely be an improvement. Now I just need to figure out how 😁